js/utility/bc-google-analytics.js

```diff
@@ -79,13 +90,102 @@ bcpl.utility.googleAnalytics = (() => {
 		addOutboundLinkTracking();
 	};
 
+	const getDefaultKey = (action) => hasOwnProperty(defaultGoogleEvents, action) ? 
+		defaultGoogleEvents[action] : 
+		null;
+
+	const getDefaultEvent = (event) => {
+		const {
+			action,
+			label
+		} = event;
+		const defaultKey = getDefaultKey(action);
+
+		return defaultKey ? 
+			{
+				[defaultKey]: label
+			} : null;
+	};
+
+	const getStandardEvent = (event) => {
+		const {
+			category,
+			label,
+			value
+		} = event;
+		const eventObj = {};
+
+		if (category) {
+			eventObj[googleEventKeys.category] = category;
+		}
+		if (label) {
+			eventObj[googleEventKeys.label] = label;
+		}
+		if (value) {
+			eventObj[googleEventKeys.value] = value;
+		}
+
+		return eventObj;
+	};
+
+	const getEventData = (event) => getDefaultEvent(event) || getStandardEvent(event);
+
+	const trackEvent = (event) => {
+		const eventData = getEventData(event);
+
+		gtag('event', event.action, eventData);
```

Comment:
i think we may want to check to see if eventData exists, if it doesn't call `gtag('event', event.action)` otherwise do what you are already doing.

Reply:
done